### PR TITLE
Loan Purpose Picklist modified

### PR DESCRIPTION
**Pull request by CRMTravis**

### CODE REVIEW

File: force-app/main/default/classes/RubiconProductDetailsController.cls [hunk 1]
```diff
@@ -119,6 +119,143 @@ public inherited sharing class RubiconProductDetailsController {
...
+  public static string[] getPicklistValuesUIApi(
+    string recordTypeId,
+    string ObjectApiName,
+    string fieldApiName
+  ) {
+    string[] piclistValues = new List<string>{};
+    string BaseUrl = system.Url.getOrgDomainUrl().toExternalForm();
+    String endPoint =
+      BaseUrl +
+      '/services/data/v47.0/ui-api/object-info/' +
```
Comment (RemarioCRM) [inline force-app/main/default/classes/RubiconProductDetailsController.cls:189]: replace 47 to 50

File: force-app/main/default/aura/RubiconProductDetailsComponent/RubiconProductDetailsComponentHelper.js [hunk 2]
```diff
@@ -96,16 +96,18 @@
   },
 
   setPickListValues: function (component) {
-    const action = component.get("c.getPickListValuesList");
+    const action = component.get("c.getPickListValues");
     action.setParams({
-      objectApiName: "Loan_Calculation_Product__c",
```
Comment (RemarioCRM) [inline force-app/main/default/aura/RubiconProductDetailsComponent/RubiconProductDetailsComponentHelper.js:101]: Loan_Calculation_Product__c contains the same as Loan_Purpose_ULR__c?,

**Review by RemarioCRM — CHANGES_REQUESTED** [review]

File: force-app/main/default/aura/RubiconProductDetailsComponent/RubiconProductDetailsComponentHelper.js (hunk 2 above)
Comment (CRMTravis) [reply force-app/main/default/aura/RubiconProductDetailsComponent/RubiconProductDetailsComponentHelper.js:101]: no

File: force-app/main/default/classes/RubiconProductDetailsController.cls (hunk 1 above)
Comment (CRMTravis) [reply force-app/main/default/classes/RubiconProductDetailsController.cls:189]: it broke when I did that

File: force-app/main/default/aura/RubiconProductDetailsComponent/RubiconProductDetailsComponentHelper.js (hunk 2 above)
Comment (RemarioCRM) [reply force-app/main/default/aura/RubiconProductDetailsComponent/RubiconProductDetailsComponentHelper.js:101]: which object you store it to?

**Pull request merged**